# Service to restore robot to default configuration

---
bool success
string msg
